os.bpmsuite.smartrouter/added/launch/bpmsuite-smartrouter.sh

```diff
@@ -17,6 +17,8 @@ function prepareEnv() {
     unset KIE_SERVER_ROUTER_PORT
     unset KIE_SERVER_ROUTER_PROTOCOL
     unset KIE_SERVER_ROUTER_URL_EXTERNAL
+    unset KIE_SERVER_ROUTER_REPO_PATH
```

Comment:
The convention we try to maintain (as much as possible) is keeping the env var names the same as the system properties, but of course upper-cased, underscored, and removal of the "org." prefix. So, if org.kie.server.router.repo is the property, then KIE_SERVER_ROUTER_REPO should be the env var. The reasoning is that if a user knows what the system property is by looking at the BxMS documentation, then they should be able to know what the corresponding env var name would be.